app/javascript/src/mocks/mockData.js

```diff
@@ -1,18 +1,3 @@
-export const itemTypes = [
-    'Toothbrush',
-    'Soap',
-    'Glitter',
-    'Bath Bomb',
-    'Shoes',
-    'Socks',
-    'Silverware',
-    'TV',
-    'Sunglasses',
-    'Peanuts',
-    'Hairbrush',
-    'Bible',
-  ];
-
 export const unassembledBoxItems = [
```

Comment:
@z3ugma do you understand what these are for?

Reply:
@maebeale they are items that can be put into boxes right? They should be the same as the `InventoryItem`s in the database?

Reply:
Yeah. I'm not sure what unassembled is in this case tho. I think it had to do with adding metadata to the list of inventory types so that if something is already added to the box, it doesn't show up in the list anymore. We definitely don't need to stick with this implementation, but probably do need to handle removing items from our json list if they've been added. Np if we want this in a separate issue.

Similarly/relatedly, once the above is handled, I'm thinking we can delete this mockData.js file.

Reply:
@maebeale  OK I think I figured it out: It looks like in `BoxDesign` you pick items to add to the box's "to-do" - all the things that should someday be put in the box.

Once the box gets to the "package" step (`reviewed_at` and `reviewed_by_id` `!=nil`), in `BoxAssembly`, you mark the items that you've actually put into the box as you go, sort of like a packing list. Maybe a separate issue to make app/javascript/src/components/box-assembly-form/BoxAssemblyForm.jsx` not pull from the `app/javascripts/mocks/mockData` in a similar vein to this issue.

Reply:
Right. For the BoxDesign phase, we're creating BoxItem data, and then at BoxAssembly, we check off each BoxItem as `added_to_box`. 

I guess we'll want to make a new issue to adapt what you've done to additionally autofilter the list of available InventoryTypes that the user can "add to the box" (aka create an associated BoxItem). (Bc they can only add each InventoryType once per Box).

That's another issue -- create the actual data when using these React forms.